pkg/executor/delete.go

```diff
@@ -223,13 +238,25 @@ func (e *DeleteExec) deleteMultiTablesByChunk(ctx context.Context) error {
 			}
 		}
 	}
-	return e.removeRowsInTblRowMap(tblRowMap)
+	return e.removeRowsInTblRowMap(ctx, tblRowMap)
 }
 
-func (e *DeleteExec) removeRowsInTblRowMap(tblRowMap tableRowMapType) error {
+func (e *DeleteExec) removeRowsInTblRowMap(ctx context.Context, tblRowMap tableRowMapType) error {
 	for id, rowMap := range tblRowMap {
 		var err error
 		rowMap.Range(func(h kv.Handle, val handleInfoPair) bool {
+			if e.ignoreErr {
+				ignored, err := checkFKIgnoreErr(ctx, e.Ctx(), e.fkChecks[id], val.handleVal)
+				if err != nil {
+					return false
```

Comment:
If return from here, will ignore the error?

Reply:
Yes. It's expected. See the implementation of `checkFKIgnoreErr` and `fkc.checkRows`. If the row breaks the foreign key constraint, a warning will be appended and the function `checkFKIgnoreErr` will return `true, nil`. If the row has a wrong format, or it meets some network issues (or other errors not related to the foreign key), it'll return the error as `false, err`.

Previously, we already have an existing implementation of `INSERT ignore`. This PR keeps the same behavior with it.